src/errors/Messages.js

```diff
@@ -101,6 +101,7 @@ const Messages = {
 
   WEBHOOK_MESSAGE: 'The message was not sent by a webhook.',
   WEBHOOK_TOKEN_UNAVAILABLE: 'This action requires a webhook token, but none is available.',
+  WEBHOOK_URL_INVALID: 'The provided webhook URL is not well formed.',
```

Comment:
Up to debate if this is better

```suggestion
  WEBHOOK_URL_INVALID: 'The provided webhook URL is not valid.',
```